### DISCUSSION TITLE
Errori SQLite3

### DISCUSSION THREAD

**Discussion by Jones1982** [indomus/forum#258] [HUB: Home Assistant]
Buona sera, ho notato già da qualche settimana, che nel registro errori c'è sempre questo:
![Cattura](https://user-images.githubusercontent.com/57654443/116927581-25700880-ac5c-11eb-89e4-1a4ad9a34db6.PNG)

Nel dettaglio:
![Cattura2](https://user-images.githubusercontent.com/57654443/116927622-302a9d80-ac5c-11eb-9cf6-9ce70f9b9a72.PNG)

Devo preoccuparmi per il database sql o avete notizie in merito?
Grazie a tutti

**Comment by snakuzzo** [comment]
Se non ti interessa lo storico...
Ferma HA
Rimuovi il file del db `home-assistant_v2.db`
Avvia HA

**Reply by Jones1982** [reply]
Ciao e grazie sempre per la risposta!
Il file da te segnalato, è quasi 1GB.
Lo storico intendi tutto quello che ho fatto (aggiornamenti compresi e "punti di ripristino")  o solo la cronologia dei sensori, ecc..?
Poiché uso una iso virtualizzata con VB (hassos_ova-4.13) aggiornata a oggi (Home Assistant OS 5.13), non vorrei dover rifare tutto. 
Grazie

**Reply by snakuzzo** [reply]
Comporta solo la perdita dello storico, non delle configurazioni. Vai tranquillo
